### ISSUE TITLE
[Issue]: Audit report should reflect most recent EC Code for project ID

### ISSUE THREAD

**Issue by ClaireValdivia** [usdigitalresponse/usdr-gost#2817]
### Why is this issue important?

sometimes the EC code will change for a project ID and this is not reflected in the audit report. 

### Current State

The audit report is currently showing the first EC code submitted for a project ID.

### Expected State

If the EC code changes for a given project ID, we want the most recent EC code to display in:
- Project Summaries V2 tab - column C (Project Expenditure Category Group) and column D (Project Expenditure Category)

### Implementation Plan
The project related fields (description, expenditure category group, expenditure category) is using the first record from the `projectRecords` array:
https://github.com/usdigitalresponse/usdr-gost/blob/main/packages/server/src/arpa_reporter/lib/audit-report.js#L293

This should be updated to:
```
        const row = {
            'Project ID': projectId,
            'Project Description': projectRecords[-1].content.Project_Description__c,
            'Project Expenditure Category Group': ec(projectRecords[-1].type),
            'Project Expenditure Category': projectRecords[-1].subcategory,
            'Capital Expenditure Amount': 0,
        };
```

**Comment by ClaireValdivia** [comment]
look into for testing - make sure it works as expected for when an ec code changes tab as well

**Comment by ClaireValdivia** [comment]
@ajhyndman I first tested this changing a project's EC code from 1.9 to 1.2 and it reflected the new EC code in the audit report as we wanted.

However, then I tried changing the EC code to 5.1. The file successfully uploaded and passed validation, but the audit report still shows the old 1.2 code.

**Comment by ajhyndman** [comment]
Fixed in #3010

**Comment by ClaireValdivia** [comment]
Confirmed that this is working in staging. 

Closing this issue out, but for future context - 
Continuing related conversation with partners about treasury report scenarios when the EC code changes as current state is that a project record is generated for the treasury report for each EC code reported in the reporting period - partners like this functionality when the ec code change happens within the same treasury template as treasury does not allow uploads with duplicate project ids, but it is harder to track duplicate project ids across different treasury templates.
